### ISSUE TITLE
SRS5 WHEP have memory leaks

### ISSUE THREAD

**Issue by hskent** [ossrs/srs#4155]
**Describe the bug**
When using WHEP for playback, memory usage increases over time.

**Version**
5.0.210

**To Reproduce**

```
listen              1935;
max_connections     1000;
srs_log_tank        file;
srs_log_file        ./logs/srs.log;
daemon              on;
http_api {
    enabled         on;
    listen          1985;
}
http_server {
    enabled         on;
    listen          8080;
    dir             ./objs/nginx/html;
}
rtc_server {
    enabled on;
    listen 8000; # UDP port
    candidate $CANDIDATE;
}
vhost __defaultVhost__ {
    min_latency     on;
    mr {
        enabled     off;
    }
    mw_latency      1000;
    http_remux {
        enabled     on;
        mount       [vhost]/[app]/[stream].flv;
    }
    rtc {
        enabled     on;
        # @see https://ossrs.net/lts/zh-cn/docs/v4/doc/webrtc#rtmp-to-rtc
        rtmp_to_rtc on;
        # @see https://ossrs.net/lts/zh-cn/docs/v4/doc/webrtc#rtc-to-rtmp
        rtc_to_rtmp off;
    }
}
```
Using the test tool:
https://github.com/Softvelum/
[whep-load-tester](https://github.com/Softvelum/whep-load-tester)

```
./whep_loader -whep-addr "http://SRS_SVR:1985/rtc/v1/whep/?app=live&stream=test" -whep-sessions 10
```

**Expected behavior**
Memory consumption increases over time as user sessions continue.

**Screenshots**
![image](https://github.com/user-attachments/assets/72a4a33c-ab1a-487c-a0c5-c83d4941b477)
![image](https://github.com/user-attachments/assets/f225c30c-84e6-4bb3-bec7-107873203350)

**Comment by winlinvip** [comment]
这个是一个已知问题，SRS 5的Source没有清理，这个问题在6.0已经解决，之前已经有一些issue说这个问题了，请不要提重复的问题。

**Comment by winlinvip** [comment]
This is a known issue where the Source in SRS 5 was not cleaned up. The issue has been resolved in version 6.0. There have already been several issues reported about this, so please refrain from submitting duplicate issues.

`TRANS_BY_GPT4`
